lib/path.js

```diff
@@ -1014,7 +1018,17 @@ const posix = {
     let resolvedAbsolute = false;
 
     for (let i = args.length - 1; i >= -1 && !resolvedAbsolute; i--) {
-      const path = i >= 0 ? args[i] : process.cwd();
+      const path = i >= 0 ?
+        args[i] :
+        (() => {
+          const _ = StringPrototypeReplace(
+            process.cwd(),
+            new RegExp(`\\${module.exports.sep}`, 'g'),
+            posix.sep
+          );
+          return StringPrototypeSlice(_, StringPrototypeIndexOf(_, posix.sep));
+        })();
```

Comment:
Can you use a top level function instead? Allocating a new closure for each loop execution is expensive.

Reply:
I'll change the ternary to an `if/else` to get rid of the closure.

Reply:
All fixed up. PTAL